front/src/Connexion/ConnectionManager.ts

```diff
@@ -72,6 +72,10 @@ class ConnectionManager {
         //Logout user in pusher and hydra
         const token = localUserStore.getAuthToken();
         await Axios.get(`${PUSHER_URL}/logout-callback`, { params: { token } }).then((res) => res.data);
+        if (this._currentRoom?.opidLogoutRedirectUrl != undefined) {
+            window.location.replace(this._currentRoom?.opidLogoutRedirectUrl);
+            return;
+        }
```

Comment:
We should probably not skip the `localUserStore.setAuthToken(null);` at the line below.

Also, instead of having a value of "opidLogoutRedirectUrl" that can be null or undefined, I'd make the property a string (compulsory) and I'd use "/" as a default value (since this is what we use today as a default value at line 82)